ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsStateStore.java

```diff
@@ -69,15 +72,14 @@ LogicalSchema schema() {
   }
 
   <T> T store(final QueryableStoreType<T> queryableStoreType) {
-    awaitRunning();
-
     try {
       if (ksqlConfig.getBoolean(KsqlConfig.KSQL_QUERY_PULL_ENABLE_STANDBY_READS)) {
         // True flag allows queries on standby and replica state stores
         return kafkaStreams.store(
             StoreQueryParameters.fromNameAndType(stateStoreName, queryableStoreType)
                 .enableStaleStores());
       } else {
+        awaitRunning();
```

Comment:
could we remove this altogether and let it error inside `.store()`?

Reply:
Done